spec/flipper/adapters/dalli_spec.rb

```diff
@@ -7,7 +7,7 @@
   let(:memory_adapter) do
     Flipper::Adapters::OperationLogger.new(Flipper::Adapters::Memory.new)
   end
-  let(:cache)   { Dalli::Client.new(ENV['MEMCACHED_URL'] || '127.0.0.1:11211') }
```

Comment:
Removing the fallback since that would also cause failure in the docker environment if used (_and_ it is the fallback Dalli [already uses](https://github.com/petergoldstein/dalli/blob/ca8b286de230546c3bfe2789e9b6bc5a2c619570/lib/dalli/client.rb#L35
)).